test/performance/odp_packet_gen.c

```diff
@@ -1378,6 +1446,32 @@ int main(int argc, char **argv)
 	/* Wait until workers have started. */
 	odp_barrier_wait(&global->barrier);
 
+	/* Periodic statistics printing */
+	if (global->test_options.update_msec) {
+		odp_time_t t1, t2;
+		uint64_t nsec;
+		int num_pktio = global->test_options.num_pktio;
+
+		printf("\n\nPackets per interface\n");
+		printf("  Dir          Time");
+		for (i = 0; i < num_pktio; i++)
+			printf(" %10i", i);
+
+		printf("\n  -----------------");
+		for (i = 0; i < num_pktio; i++)
+			printf("-----------");
+
+		printf("\n");
+
+		t1 = odp_time_local();
+		while (odp_atomic_load_u32(&global->exit_test) == 0) {
+			usleep(1000 * global->test_options.update_msec);
+			t2 = odp_time_local();
+			nsec = odp_time_diff_ns(t2, t1);
+			print_periodic_stat(global, nsec);
+		}
+	}
```

Comment:
I would prefer having all printing code inside `print_periodic_stat()` function. The PR looks good othwewise.

Reply:
fixed in v3